templates/base.html

```diff
@@ -186,21 +186,56 @@
         }
       }
 
-      const versionResetForm = (event) => {
-        if (event.persisted) {
-          const form = document.getElementById('id_version').closest('form');
-          if (form) {
-            form.reset();
+      const changeVersionAndCategory = (event) => {
```

Comment:
Interesting approach! Perhaps a comment showing a couple of examples of what this is doing would be helpful. It took me a while to grok it.